src/server/index.jsx

```diff
@@ -126,9 +133,13 @@ if (process.env.SIMORGH_APP_ENV === 'local') {
 }
 
 const injectDefaultCacheHeader = (req, res, next) => {
+  const defaultMaxAge = getDefaultMaxAge(req);
+  const maxAge = req.originalUrl.indexOf('/topics/') !== -1 ? defaultMaxAge * 2 : defaultMaxAge;
   res.set(
-    'cache-control',
-    `public, stale-if-error=90, stale-while-revalidate=30, max-age=30`,
+    'Cache-Control',
+    `public, stale-if-error=${
+      maxAge * 4
```

Comment:
Is the `*4` needed here?

Reply:
I thought we'd multiply whichever maxAge by 4 in order to have a longer cache when the server responds with a non-200 response

e.g. for 45s we'd serve stale for 180s

Does that make sense or have I gone off the rails here?

Reply:
Ah yea makes sense. I read that as the value for `max-age`. Linter taking new lines confused me!

Reply:
Just some lint fixes needed

Reply:
grand. Thanks

I've run the idea passed Neil and he's happy with it, so I'll tidy this up and then we can launch

Reply:
Launching tomorrow so we can see any effects from today's deployments